vars/seleniumExecuteTests.groovy

```diff
@@ -39,7 +39,8 @@ import groovy.text.GStringTemplateEngine
      */
     'gitBranch',
     /**
-     * Only if `testRepository` is provided: Credentials for a protected testRepository
+     * If `testRepository` is provided: Credentials for a protected testRepository.
+     * Also used for sshagent to allow running npm install with github dependencies.
```

Comment:
So the same credential can be used for two different things? Could that lead to a conflict: What if someone wants to configure a credential for both a protected 'testRepository' **and** for npm install with github dependencies? Or is that the same thing? Maybe I'm not getting it, and maybe some unit test(s) could improve my understanding... ;)